src/audio/asrc/asrc.c

```diff
@@ -535,9 +535,9 @@ static int asrc_prepare(struct comp_dev *dev)
 	struct comp_buffer *sourceb;
 	uint32_t source_period_bytes;
 	uint32_t sink_period_bytes;
-	int sample_bytes;
-	int sample_bits;
-	int frame_bytes;
+	int sbytes;
```

Comment:
I don't actually like the change to less-descriptive names. If it's for alignment, I would rather you do the change the other way, the sof/audio/format.h functions themselves being changed to be more descriptive if reasonable.

Reply:
Yes that's a good argument. Maybe prefixing the two functions with "get_" is better.

Reply:
Done, thanks for suggestion!